langlib/lang.array/src/main/java/org/ballerinalang/langlib/array/Next.java

```diff
@@ -49,16 +59,34 @@
     //TODO: refactor hard coded values
     public static Object next(Strand strand, ObjectValue m) {
         IteratorValue arrIterator = (IteratorValue) m.getNativeData("&iterator&");
-
+        ArrayValue arr = (ArrayValue) m.get("m");
```

Comment:
`get(String fieldName)` seems to be deprecated?

Reply:
will fix it.